bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/airqBindingConstants.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.airq.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link airqBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Aurelio Caliaro - Initial contribution
+ */
+@NonNullByDefault
+public class airqBindingConstants {
+
+    private static final String BINDING_ID = "airq";
+
+    // List of all Thing Type UIDs
+    // public static final ThingTypeUID THING_TYPE_SAMPLE = new ThingTypeUID(BINDING_ID, "sample");
+    public static final ThingTypeUID THING_TYPE_AIRQ = new ThingTypeUID(BINDING_ID, "airq");
+
+    // List of all Channel ids
+    public static final String CHANNEL_1 = "channel1";
```

Comment:
Remove these unless you do have a channel1 ?... If you had used this example to define all the channel names, it would have been easier when you changed their name due to not using mixedCase naming. Most bindings do this.

Reply:
I will do that in another binding I am developing, but I won't change it in this one. The issue is anyway changing the names in three files: airqHandler.java, README.md and thing-types.xml. The constant only helps for the .java file, afaik.

Reply:
Yes only the java file, but if you have 10 java files and they are used in multiple places in each it can be a HUGE pain.